test/addr_manager_suite.go

```diff
@@ -0,0 +1,280 @@
+package test
```

Comment:
Could we make the naming of the file consistent with the new naming of the interface?

Reply:
Totally, good catch!